antarest/launcher/adapters/local_launcher/local_launcher.py

```diff
@@ -168,33 +168,53 @@ def stop_reading_output() -> bool:
                 subprocess.run(["Rscript", "post-processing.R"], cwd=export_path)
 
             output_id: Optional[str] = None
-            try:
-                output_id = self.callbacks.import_output(str(uuid), export_path / "output", {})
-            except Exception as e:
-                logger.error(
-                    f"Failed to import output for study {study_uuid} located at {export_path}",
-                    exc_info=e,
-                )
-            del self.job_id_to_study_id[str(uuid)]
+            if process.returncode == 0:
```

Comment:
In slurm laucher we still call the import output when it has failed, we need to understand if it makes sense

Reply:
From my understanding, if the simulator fails, the output is not retrieved from calin, therefore the `_import_output` fails. But with the local launcher the output exists and therefore it can be retrieved. I think we should keep the code as it is, else i would have to touch code shared with the slurm part

Reply:
Ok. Indeed if we still wanted to import the output, we should have some kind of status to say that it's a "failed output".

I wonder if we should not also delete the import output in slurm implementation, to clarify the behaviour ?